src/Microsoft.Health.Fhir.Api/Features/Throttling/ThrottlingMiddleware.cs

```diff
@@ -33,19 +32,25 @@ public sealed class ThrottlingMiddleware : IAsyncDisposable, IDisposable
         private const double RetryAfterDecayRate = 1.1;
         private const int SamplePeriodMilliseconds = 500;
 
+        // hard-coding these to minimize resource consumption when throttling
+        private const string ThrottledContentType = "application/json; charset=utf-8";
+        private static readonly ReadOnlyMemory<byte> _throttledBody = Encoding.UTF8.GetBytes($@"{{""severity"":""Error"",""code"":""Throttled"",""diagnostics"":""{Resources.TooManyConcurrentRequests}"",""location"":null}}").AsMemory();
```

Comment:
nit: perhaps remove `""location"":null` since it is null, and also deprecated in R4

Reply:
Done